src/Common/src/Interop/Windows/NCrypt/Interop.Properties.cs

```diff
@@ -16,5 +17,32 @@ internal static partial class NCrypt
 
         [DllImport(Interop.Libraries.NCrypt, CharSet = CharSet.Unicode)]
         internal static extern unsafe ErrorCode NCryptSetProperty(SafeNCryptHandle hObject, string pszProperty, [In] void* pbInput, int cbInput, CngPropertyOptions dwFlags);
+
+        internal static ErrorCode NCryptGetIntProperty(SafeNCryptHandle hObject, string pszProperty, ref int result)
+        {
+            int cbResult;
+            ErrorCode errorCode;
+
+            unsafe
+            {
+                fixed (int* pResult = &result)
+                {
+                    errorCode = Interop.NCrypt.NCryptGetProperty(
+                        hObject,
+                        pszProperty,
+                        pResult,
+                        sizeof(int),
+                        out cbResult,
+                        CngPropertyOptions.None);
+                }
+            }
+
+            if (errorCode == ErrorCode.ERROR_SUCCESS)
+            {
+                Debug.Assert(cbResult == sizeof(int));
+            }
```

Comment:
I will leave as-is as an errorcode is expected in some cases (when the property doesn't exist)